include/revng/Pipeline/Context.h

```diff
@@ -36,6 +36,7 @@ extern Logger<> CommandLogger;
 class Context {
 private:
   GlobalsMap Globals;
+  uint64_t GlobalIndex = 0;
```

Comment:
Calling this feature "global index" is very vague.
What about "commit index" or "epoch"?

Reply:
[The pad](https://pad.rev.ng/NSpY-VvKTnG23vjTDJO70g) explicitly forbid using 'epoch'. I guess 'commit index' works

Reply:
Yup

Reply:
Done